apps/frontend/src/components/Landing.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Nav from "./Nav";
+import { Navigate, useNavigate } from "react-router-dom";
+
+type Props = {};
+
+const Landing = (props: Props) => {
```

Comment:
Both Navigate and props are not used and can be safely removed

```suggestion
import { useNavigate } from "react-router-dom";

const Landing = () => {
```

Reply:
I've deleted it